recipes/pandasql/meta.yaml

```
@@ -0,0 +1,45 @@
+{% set name = "pandasql" %}
+{% set version = "0.7.3" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
+  sha256: 1eb248869086435a7d85281ebd9fe525d69d9d954a0dceb854f71a8d0fd8de69
+
+build:
+  number: 0
```

This looks like it can be `noarch: python`:

```suggestion
  noarch: python
  number: 0
```